src/rules/pyupgrade/snapshots/ruff__rules__pyupgrade__tests__UP035_UP035.py.snap

```diff
@@ -0,0 +1,260 @@
+---
+source: src/rules/pyupgrade/mod.rs
+expression: diagnostics
+---
+- kind:
+    ImportReplacements: ~
+  location:
+    row: 2
+    column: 0
+  end_location:
+    row: 2
+    column: 31
+  fix:
+    content: from collections.abc import Mapping
```

Comment:
The snap format has changed (slightly) to accommodate newlines. Should be autogenerated after a rebase and test run